recipe/meta.yaml

```diff
@@ -62,6 +60,7 @@ outputs:
         - scikit-posthocs >=0.6.5
         - seaborn >=0.11.0
         - statsforecast >=0.5.2
+        - statsmodels >=0.12.1
         - stumpy >=1.5.1
         - tbats >=1.1.0
         - tsfresh >=0.17.0
```

Comment:
A few dependencies in run_constrained use python selector in the upstream. Do we need that here?

Reply:
See my comment above